test/indent_anchor_test.rb

```diff
@@ -0,0 +1,737 @@
+# frozen_string_literal: true
+
+require_relative 'lib'
+
+def check_difference_oppen_wadler(margin, expected_oppen, expected_wadler, builder_block)
+  printer = Oppen::Wadler.new(margin:, config: Oppen::Config.oppen)
+  builder_block.call(printer)
+  _(printer.output).must_equal expected_oppen, 'Oppen failed the test'
+
+  printer = Oppen::Wadler.new(margin:, config: Oppen::Config.wadler)
+  builder_block.call(printer)
+  _(printer.output).must_equal expected_wadler, 'Wadler failed the test'
+end
+
+describe 'Generated anchor tests' do
+  [
+    [
+      'must work with a simple group of indentation 0',
+      proc { |out|
+        out.group(0) {
+          out.text 'Hello, World!'
+          out.break
+          out.text 'How are you?'
+          out.break
+          out.text 'I am fine and you?'
+        }
+      }, <<~EXPECTED_OPPEN.chomp, <<~EXPECTED_WADLER.chomp, [30]
+        Hello, World!
+        How are you?
+        I am fine and you?
+      EXPECTED_OPPEN
+        Hello, World!
+        How are you?
+        I am fine and you?
+      EXPECTED_WADLER
+    ],
```

Comment:
```suggestion
      }, <<~OPPEN.chomp, <<~WADLER.chomp, [30]
        Hello, World!
        How are you?
        I am fine and you?
      OPPEN
        Hello, World!
        How are you?
        I am fine and you?
      WADLER
    ],
```

If anything, but we really want them to coincide, so why repeat them? Just `EXPECTED` and it should be a single test.

Reply:
They almost never coincide, this is why I have 2 expected